gallery/src/styles/NxList/NxListClickableExample.tsx

```diff
@@ -10,31 +10,41 @@ import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 
 const NxListClickableExample = () =>
   <ul className="nx-list nx-list--clickable">
-    <li className="nx-list__item" tabIndex={0}>
-      <span className="nx-list__text">Action 1</span>
-      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    <li className="nx-list__item nx-list__item--clickable">
```

Comment:
The top border here is thicker than it used to be

Reply:
> The top border here is thicker than it used to be

Yeah, I noticed that too, fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/24e0af1dec552b42d7c08a2b7cc8a3a486359a52

Reply:
> Yeah, I noticed that too, fixed here: 24e0af1

Pretty sure that's wrong.  I think you were trying to remove the border from the first list item there but you're actually removing the top border on the list itself, when the list is the first item within its parent.

Reply:
> > Yeah, I noticed that too, fixed here: [24e0af1](https://github.com/sonatype/sonatype-react-shared-components/commit/24e0af1dec552b42d7c08a2b7cc8a3a486359a52)
> 
> Pretty sure that's wrong. I think you were trying to remove the border from the first list item there but you're actually removing the top border on the list itself, when the list is the first item within its parent.

Oops, got my indentation wrong: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/dab89e9c8dc90fc2c1be4ef0db1b3746fd3c27f0